### PR TITLE
Validation: Description-missing fixes.

### PR DESCRIPTION
**Pull request by brittmsantos**
Added missing descriptions to fix validation errors.

### CODE REVIEW

**Comment by brittmsantos** [comment]
@stevemunk @rbrundritt

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: BingMaps/v8-web-control/modules/spatial-data-service-module/geodataresult-object.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "GeoDataResult Object | Microsoft Docs"
+description: Describes the GeoDataResult object, which represents a single result GeoData API object, and provides its properties.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/modules/spatial-data-service-module/geodataresult-object.md:3]:
```suggestion
description: Describes the GeoDataResult object, which represents a single result GeoData API object, and includes descriptions for each of its properties.
```

File: BingMaps/v8-web-control/modules/spatial-data-service-module/getboundaryrequestoptions-object.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "GetBoundaryRequestOptions Object | Microsoft Docs"
+description: Describes the GetBoundaryRequestOptions object, which contains the options for GeoData API requests boundary data, and provides its properties.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/modules/spatial-data-service-module/getboundaryrequestoptions-object.md:3]:
```suggestion
description: Describes the GetBoundaryRequestOptions object, which contains the options for GeoData API requests boundary data, with descriptions for each of its properties.
```

File: BingMaps/v8-web-control/modules/spatial-data-service-module/index.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Spatial Data Service Module | Microsoft Docs"
+description: Describes the Spatial Data Service module, which allows uploading data to expose as a spatial REST service for applications.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/modules/spatial-data-service-module/index.md:3]:
```suggestion
description: The overview page for the Spatial Data Service module, which allows uploading data to expose as a spatial REST service for applications, with links to articles that describe the various classes, enumerators, a filter example and several more examples demonstrating both the Query and GeoData API.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by brittmsantos** [comment]
@stevemunk 

I've implemented the requested changes. Please let me know if there are any other issues preventing approval and merge. Thank you and have a great day!

**Comment by stevemunk** [comment]
@brittmsantos Thank you! Merging....

**Pull request merged**